src/app/containers/ChartbeatAnalytics/canonical/index.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
 import { mount } from 'enzyme';
+import renderWithHelmet from '@bbc/psammead-test-helpers/dist/renderWithHelmet';
```

Comment:
since we're now using `renderWithHelmet`, i'd make a plea to add something in the [readme](https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-test-helpers/README.md#exported-functions) of how this works and it's usage on [this other pr](https://github.com/bbc/psammead/pull/2405)